libs/category/src/models/filters/category-filter.ts

```diff
@@ -0,0 +1,10 @@
+import { DaffCategoryFilterEqual } from './type/equal/filter';
+import { DaffCategoryFilterRangeNumeric } from './type/range-numeric/filter';
+
+/**
+ * A single category filter, along with it's options. For example, a filter may be
+ * something like "Color", and have the filter options of "Blue", "Green,", etc.
+ * Additionally, filters have "types", see {@link DaffCategoryFilterType} for a
+ * complete list.
```

Comment:
Comma splice. Should be two sentences or reworded

Reply:
5ba30237

Reply:
Talking about this sentence: "Additionally, filters have "types", see {@link DaffCategoryFilterType} for a complete list."
Comma splice between "types" and "see".

Reply:
cf0d5c4f